### PR TITLE
Newsletter form added

### PR DESCRIPTION
**Pull request by AngryKitten91**
Added newsletter form and simple e-mail address validation.

### CODE REVIEW

File: css/components/_components.cookie.scss [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
+    text-align: center;
     justify-content: space-around;
 }
 
 button.o-cookie-hide{
-    margin: 0;
-    padding: 5px;
-    border: 1px solid $color3;
-    border-radius: 10px;
+    // margin: 0;
```
Comment (Omfalos) [inline css/components/_components.cookie.scss]: delete if it is not need thsi way you will have only needed code here

File: js/script.js [hunk 2]
```diff
@@ -81,21 +81,51 @@ document.addEventListener("DOMContentLoaded", function (event) {
...
-    function cookieOff() {
-        if (localStorage.getItem('state') === 'on') {
-            localStorage.setItem('state', 'off');
-            var cookieBox = document.querySelector('.o-cookie-info');
-            cookieBox.classList.add('o-cookie-info--hidden');
+function formValidate() {
+    let x = document.forms["form"]["emails[]"].value;
+    let formInfo = document.getElementById('js-formInfo');
+    let emailCheck = validateEmail(x);
+    if (emailCheck === true) {
```
Comment (Omfalos) [inline js/script.js]:
```javascript
if (!emailCheck) {
if (x == "") {
            formInfo.textContent= 'Proszę wprowadź adres e-mail';
            formInfo.classList.remove('u-hidden');
        } else {
            formInfo.textContent= 'Proszę wprowadź poprawny adres e-mail';
            formInfo.classList.remove('u-hidden');
        }
 return emailCheck
}  
 formInfo.classList.add('u-hidden');
return emailCheck;
```

File: css/components/_components.cookie.scss (hunk 1 above)
Comment (AngryKitten91) [reply css/components/_components.cookie.scss]: @Omfalos Removed unnecessary commented lines

File: js/script.js (hunk 2 above)
Comment (AngryKitten91) [reply js/script.js]: @Omfalos Code replaced with suggested snippet ;)

**Pull request merged**